libs/README.md

```diff
@@ -42,6 +42,23 @@ Although published to package managers, the following libraries were developed f
 | [@hashintel/design-system]       | TypeScript  | [npm](https://www.npmjs.com/package/@hashintel/design-system)       | To be written | A collection of styleguide-aligned reusable UI primitives for [HASH] and our [hash.ai] website       |
 | [@hashintel/block-design-system] | TypeScript  | [npm](https://www.npmjs.com/package/@hashintel/block-design-system) | To be written | A relatively unopinionated set of reusable UI primitives for use in building [Block Protocol] blocks |
 
+## Internal unpublished libraries
+
```

Comment:
Maybe we don't need that heading here?
```suggestion
```
Or maybe making it smaller?
```suggestion
# Internal unpublished libraries

```

Reply:
I'd be happy with...

```suggestion
## Internal libraries

```

...the text beneath makes clear they are unpublished. I think the key thing to highlight anyway is that these are internal, and not intended to be relied on more widely.

Reply:
The section above is already called "Internal libraries" 😄